patches/api/0312-Missing-Entity-Behavior-API.patch

```diff
@@ -695,6 +695,24 @@ index 479f7a7c54c85cb685f56e60906650d1989c03ff..60267ee382de80fab86b440ff72a2455
 +     * @param standing whether the polar bear should be standing
 +     */
 +    void setStanding(boolean standing);
++
++    /**
++     * Gets when the polar bear will play its next
++     * warning sound when attacking an entity.
++     *
++     * This will tick down until it reaches 0, will it will
```

Comment:
Will it will? :D

Reply:
Lol thanks, alot of these javadocs are unfinished as I still need to test the exact behavior of some of these methods.
But feel free to point out any of these, 
thank you!